src/core_modules/capture-core/components/D2Form/field/configs/base/configBase.js

```diff
@@ -1,4 +1,5 @@
 // @flow
+import React from 'react';
```

Comment:
Even better:
```suggestion
import * as React from 'react';
```
We need the * as React to get the ComponentType type